Gemfile.lock

```diff
@@ -605,7 +601,6 @@ GEM
 PLATFORMS
   arm64-darwin-21
   arm64-darwin-22
-  ruby
```

Comment:
@newstler, I'm curios about all the removals in `Gemfile.lock`. Was that intentional?

Reply:
Most probably not, the only thing that should have changed there is update of `jbuilder-schema` to `2.2.0`. I just restored Gemfile.lock from `main`, updated `jbuilder-schema` again and rebased everything against `main`. Doesn't fail on my local machine

Reply:
👍 Excellent! The `Release:` failures here are just saying that these changes would break against the versions of the `core` gems that are currently pinned in `Gemfile.lock`. I'll merge the related PR in `core`, then release a new version of the gems, and bump `Gemfile.lock` and then this should be all good. Thanks for your work on it!